tests/wpunit/TEC/Events/Calendar_Embeds/Calendar_EmbedsTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace TEC\Events\Calendar_Embeds;
+
+use TEC\Events\Calendar_Embeds\Calendar_Embeds;
+use Tribe\Tests\Traits\With_Uopz;
+use Tribe__Events__Main as TEC;
+
+class Calendar_EmbedsTest extends \Codeception\TestCase\WPTestCase {
+	use With_Uopz;
+
+	/**
+	 * @var Calendar_Embeds
+	 */
+	protected $calendarEmbeds;
+
+	protected function setUp(): void {
+		parent::setUp();
+		$this->calendarEmbeds = new Calendar_Embeds();
+	}
+
+	protected function tearDown() : void {
+		parent::tearDown();
+		$this->calendarEmbeds = null;
+		$this->unset_uopz_returns();
+	}
```

Comment:
Both of these methods are not needed.

Also you are binding the Calendar_Embeds to the container as a singleton and then you go ahead and create a new instance, invaliding test cases since they no longer mimic production code.

Reply:
Expanding because i didnt really explain my point.

When we need to grab the instance of CalendarEmbed we should do it though our container - since this is going to become a controller test case soon we will have test_services available!

the unset_uopz_returns should be called regardless after every test case.